pytradfri/api/aiocoap_api.py

```diff
@@ -175,14 +188,46 @@ def psk(self, value):
 
         def success_callback(res):
             api_command.result = _process_output(res)
+            APIFactory.update_last_changed()
 
         def error_callback(ex):
             err_callback(ex)
 
         ob = pr.observation
         ob.register_callback(success_callback)
         ob.register_errback(error_callback)
-        self._observations_err_callbacks.append(ob.error)
+        self._observations.append(ob)
+
+    async def _check_observations(self):
```

Comment:
Method name needs improving as this isn't just checking for dead observations, please also add a docstring.

Reply:
Ammended in 3fb5c87f231d931ae531b917bc0f1cb132a6f118